source/config.js

```diff
@@ -148,6 +148,9 @@ const defaultConfig = {
 
   // Array of local git repo paths to display at the ungit home page
   defaultRepositories: [],
+
+  // is force ungit server to bind to `127.0.0.1`
+  isBindToLocalhost: true,
```

Comment:
Why not using a string?
If someone has multiple networkinterfaces you can bind to a specific one.

Reply:
you are right, will touch that up soon.